### PR TITLE
fix: remove auth connector from ethers and check for socials length

### PR DESCRIPTION
**Pull request by tomiir**
# Description
Auth connector was being injected in ethers even when setting `email: false`

## Type of change

- [ ] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

For Linear issues: Closes APKT-874

# Checklist

- [] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [X] My changes generate no new warnings
- [X] I have reviewed my own code
- [X] I have filled out all required sections

### CODE REVIEW

File: packages/wagmi/src/utils/defaultWagmiCoreConfig.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ export function defaultWagmiConfig({
     ...auth
   }
 
-  if (mergedAuth.email || mergedAuth.socials) {
+  if (mergedAuth.email || mergedAuth.socials?.length) {
```
Comment (enesozturk) [inline packages/wagmi/src/utils/defaultWagmiCoreConfig.ts:79]: Why don't we allow users to have email login if they don't set social options? Cannot they use only email?

**Review by enesozturk — APPROVED** [review]

**Review by svenvoskamp — APPROVED** [review]

File: apps/laboratory/src/utils/DataUtil.ts [hunk 2]
```diff
@@ -61,6 +66,11 @@ export const ethersSdkOptions = [
     title: 'Email and Social',
     link: '/library/ethers-email/',
     description: 'Configuration using ethers and implementing email and social login'
+  },
+  {
```
Comment (enesozturk) [inline apps/laboratory/src/utils/DataUtil.ts:70]: What about `ethers5` no email?

Comment (enesozturk) [reply apps/laboratory/src/utils/DataUtil.ts:70]: We could even test solana since it's enabled for Solana as well

Comment (tomiir) [reply apps/laboratory/src/utils/DataUtil.ts:70]: Removing ethers5 email until we get it working properly

Let me change this for solana

Comment (tomiir) [reply apps/laboratory/src/utils/DataUtil.ts:70]: It doesn't really have a working email tbh. LEaving it out until we have it stable. 
Adding solana too 🫡

**Pull request merged**